arch/x86/include/asm/page_64_types.h

```diff
@@ -68,7 +68,7 @@
  * If KASLR is disabled we can shrink it to 0.5 GiB and increase the size
  * of the modules area to 1.5 GiB.
  */
-#ifdef CONFIG_RANDOMIZE_BASE
+#if 1
```

Comment:
Leo, can you keep this change local? It's not something that we'll eventually want to merge.